crates/tests/tedge_test_utils/src/fs.rs

```diff
@@ -88,7 +88,7 @@ impl TempTedgeFile {
         file.sync_all().unwrap();
     }
 
-    pub fn with_toml_content(self, content: toml::Value) {
+    pub fn with_toml_content(self, content: toml::Table) {
```

Comment:
What is the motivation for this change? The former version were more generic.

Reply:
In the toml `0.7` version, the `toml` macro builds the `Table`. So, I had to update this.